Dockerfile

```diff
@@ -15,11 +15,7 @@ WORKDIR /usr/src/app
 COPY requirements.txt /usr/src/app
 RUN pip install -r requirements.txt
 
-# Installing secrets_safe library
```

Comment:
I do not see workflows that can be affected by this change, repo only have dependabot.yml, Could you please point me out what part can be affected ?

Reply:
Adding the my-secrets-safe-library==2.0 on the requirements.txt installs the secrets safe library on the Docker image?

Reply:
Yes

Reply:
Ok, let¿s comment the lines on the Dockerfile in case they are needed because the secret safe library is not on a official beyond trust pypi repository yet.

Reply:
Sure